src/App.tsx

```diff
@@ -53,6 +54,7 @@ export default function ChatApp() {
         width: "100%",
       }}
     >
+      <Presence />
```

Comment:
Probably put this in the Header component itself, and on the right corner of the screen